website/docs/developer/development-guide/code/unit-tests.md

```diff
@@ -0,0 +1,17 @@
+---
+id: unit-tests
+title: Running Unit Tests
+description: Instructions on running unit tests.
+---
+
+## Running Unit Tests and Coverage Checks
+1. Navigate to the task's base folder.
+1. Activate the virtual environment.
+1. Run
+    ```commandline
+    coverage run --source [name of lambda] -m pytest
+    ```
+1. Output the coverage results to the file system by running
+    ```commandline
+    coverage html
+    ```
```

Comment:
Maybe put some comments in that we expect at least 80% coverage from uint tests and that all unit-test pass. Also, all new code must have a unit test.

You may also want to put some notes in about the test harness we are using `pytest` and  expectations with some links. I would also add any code/styling standards for the tests we may want to ask for/enforce for the PR.

Reply:
Updated command, and added link.

Reply:
Sounds good. I'll get started on a unit-test dev guide section.

Reply:
Added something